glusterd2/transaction/transaction.go

```diff
@@ -125,12 +127,14 @@ func (t *Txn) Do() error {
 		return err
 	}
 
+	var origCtx []context.Context
```

Comment:
You can do this in one step by `origCtx := []context.Context{t.OrigCtx}`

Reply:
With the latest change, this change is no longer necessary.